internal/pkg/cli/flag.go

```diff
@@ -217,7 +217,9 @@ Tasks with the same group name share the same set of resources.
 Accepts valid Go duration strings. For example: "2h", "1h30m", "900s".`
 	scheduleFlagDescription = `The schedule on which to run this job. 
 Accepts cron expressions of the format (M H DoM M DoW) and schedule definition strings. 
-For example: "0 * * * *", "@daily", "@weekly", "@every 1h30m".`
+For example: "0 * * * *", "@daily", "@weekly", "@every 1h30m".
+AWS Schedule Expressions of the form "rate(10 minutes)" or "cron(0 12 L * ? 2021)"
+are also accepted.`
```

Comment:
nit: Should this more closely parallel the wording/capitalization in the preceding lines? ("AWS schedule expressions are also accepted. For example: "rate(10 minutes)" or "cron(0 12 L * ? 2021)"" or something like that?)